src/caesar-cipher.js

```diff
@@ -0,0 +1,19 @@
+const CaesarCipher = (() => {
```

Comment:
Not 100% sure about the decision you took to capitalize here, it is not a class nor a constructor.